flang/lib/Optimizer/CodeGen/Target.cpp

```diff
@@ -173,6 +173,7 @@ fir::CodeGenSpecifics::get(mlir::MLIRContext *ctx, llvm::Triple &trp,
     default:
       break;
     case llvm::Triple::OSType::Linux:
+    case llvm::Triple::OSType::Darwin:
```

Comment:
I wonder how many 32 bit macs are out there...

Reply:
Probably none. I'm happy to remove this if you prefer. I just added it for uniformity with Linux.